### PR TITLE
Tina/visuals Various style changes to match Figma prototype design

### PR DESCRIPTION
**Pull request by QueenChristina**
Creating pull request because newly discussed TODOs are focused on new features, better suited on new branches, and since majority of preliminary visuals done.

Features:

- various design and style changes to match Figma design
- various alignment and spacing/padding changes
- responsiveness (slider orientation, font sizes, etc.)
- Unify font to Lato
- Fix Recommendation card height to be constant
- Add scroll bar to Recommendation card if text too long
- keep keyboard focus on search bar after "Add" food
- Go to homepage if click logo
- fix gradient background to be fixed

To be clear, this branch did NOT address:

- alignment of bar chart labels or bar widths
- Clear grocery list if click home (logo)
- newly discussed features
- new vertical slider design (discussed as of 8/3/21)
- Slowness of searching
- Keep certain headers and titles on one line (auto adjust font-size)

Known issues:

- marks on slider are off center, mobile only (present since dev)
- scroll bar in Recommendation card is still NOT visible on iPhone, Safari

### CODE REVIEW

**Comment by tonychen2** [comment]
For the known issues, do we have a solution envisioned already? or do we still need to look into this?

**Comment by QueenChristina** [comment]
> For the known issues, do we have a solution envisioned already? or do we still need to look into this?

For the known issues
1) My thought is the issue might be difference in pixel density so using relative units instead of fixed pixel length to recenter the marks might fix it. Not 100% sure though, I have a few doubts about it, and the solution isn’t ideal.
2) Still need to look into it.

File: frontend/src/App.js [hunk 1]
```diff
@@ -180,21 +184,87 @@ class App extends Component {
         const barSize = 8;
         const recoSize = 4;
 
-        let headline = (this.state.hasSearched ? "My food impact" : "Track the climate impact of my food")
+        let headline = (this.state.hasSearched ? "\u2014 My food impact \u2014" : "Track the climate impact of my food")
+
+        // Override global themes for Typography. TODO: place in separate imported doc. like index.css
+        const theme = createMuiTheme({
+            typography: {
```
Comment (tonychen2) [inline frontend/src/App.js:226]: I made a small change here to the order and committed to your branch @QueenChristina

**Review by tonychen2 — APPROVED** [review]
I skimmed through the code and it looks good to me! Will just merge and continue to QA visually in our QA environment.

File: frontend/src/App.js (hunk 1 above)
Comment (QueenChristina) [reply frontend/src/App.js:226]: Awesome. Thanks for looking into it.

**Pull request merged**